### ISSUE TITLE
Mesh output?

### ISSUE THREAD

**Issue by CyberLayer-ai** [ashawkey/torch-ngp#78]
Is it possible to add an option to get a mesh output as seen in standard instant-ngp? Thank you!!

**Comment by ashawkey** [comment]
@CyberLayer-ai Yes, in the GUI there is a `mesh` button, and in the `main_*` script there is a [commented line](https://github.com/ashawkey/torch-ngp/blob/main/main_nerf.py#L151) to output mesh (disabled by default). You may need to adjust the density threshold to obtain a good mesh.
